hack/deploy-karmada.sh

```diff
@@ -103,17 +103,8 @@ function generate_cert_secret {
 # install Karmada's APIs
 function installCRDs() {
     kubectl apply -f "${REPO_ROOT}/artifacts/deploy/namespace.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/cluster.karmada.io_clusters.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/policy.karmada.io_propagationpolicies.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/policy.karmada.io_clusterpropagationpolicies.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/policy.karmada.io_overridepolicies.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/policy.karmada.io_clusteroverridepolicies.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/policy.karmada.io_replicaschedulingpolicies.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/work.karmada.io_works.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/work.karmada.io_resourcebindings.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/work.karmada.io_clusterresourcebindings.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/multicluster.x-k8s.io_serviceexports.yaml"
-    kubectl apply -f "${REPO_ROOT}/charts/_crds/bases/multicluster.x-k8s.io_serviceimports.yaml"
+
+    kustomize build "${REPO_ROOT}/charts/_crds" | kubectl apply -f -
```

Comment:
Is it better to use `kubectl kustomize`?  `kustomize` need to install separately

Reply:
Wow! I didn't know `kustomize` has integrated into kubectl.
That's great!!!

Reply:
Um, it's integrated since kubectl v1.14

Reply:
Done.